webhooks.md

```diff
@@ -0,0 +1,160 @@
+# PayrollHero Webhooks
+
+PayrollHero has a webhook system available to notify about various events in an employee's lifecycle.
+
+## Data Objects
+
+The Webhook Interface makes use of standardized payload objects to make sure that the responses are easily handled.
+
+These are:
+
+### Employee Object
+
+```json
+{
+    "id": 1,
+    "account_id": 2,
+    "phone_numbers": [
+      {
+        "label": "main",
+        "number": "+1 123-223-3343"
+      }
+    ],
+    "fullname": "John Doe"
+}
+```
+
+### Shift Object
+```json
+{
+    "clock_in_time": "2016-07-15T19:28:25+00:00",
+    "clock_out_time": "2016-07-15T19:28:25+00:00",
+    "earliest_clock_in_time": "2016-07-15T19:28:25+00:00",
```

Comment:
We'll yes and no.

You'll lose one detail which when is the user allowed to clock in? Won't you want to optimize on that instead of assuming the user could clock in 30 minutes before and show the clocking app only to have your user fail to clock in.

I do agree the name of the field is unclear. But it's not totally useless.

I'm okay either way.